c/meterpreter/source/metsrv/server_transport_winhttp.c

```diff
@@ -303,26 +302,32 @@ static DWORD packet_transmit_http(Remote *remote, LPBYTE rawPacket, DWORD rawPac
 		hReq = ctx->create_req(ctx, FALSE, "PACKET TRANSMIT");
 		if (hReq == NULL)
 		{
+			result = GetLastError();
 			break;
 		}
 
-		result = ctx->send_req(ctx, hReq, rawPacket, rawPacketLength);
-
-		if (!result)
+		res = ctx->send_req(ctx, hReq, rawPacket, rawPacketLength);
+		if (!res)
 		{
-			dprintf("[PACKET TRANSMIT] Failed HttpSendRequest: %d", GetLastError());
-			SetLastError(ERROR_NOT_FOUND);
+			result = GetLastError();
```

Comment:
```
			BREAK_ON_ERROR
```
For context https://github.com/rapid7/metasploit-payloads/blob/6e08d1f9812aa4d7a76b451fd5e0bae03975bb91/c/meterpreter/source/extensions/priv/service.c#L15-L27

Reply:
Added BREAK_ON_ERROR where appropriate!